docs/installation/application.rst

```diff
@@ -0,0 +1,196 @@
+  	dataset:
+  	spatial:
+
+**services.yml Options:**
+
+* **version**: Indicated the version of the :file:`services.yml` file. Current default : 1.0
+* **skip**: If enabled, it will skip the installation of services. This option is set to `False` by default. 
+* **interactive**: If enabled, it will start an interactive mode that will let you select from existing portal services. This option is set to `False` by default. 
+
+
+* **persistent** : List of persistent store settings in the app and the service to link to each.
+* **dataset** : List of dataset settings in the app and the service to link to each.
+* **spatial** : List of spatial persistent store settings in the app and the service to link to each.
+* **wps** : List of Web Processing service settings in the app and the service to link to each. 
+
+Settings in each of the service sections above will need to be listed in the following format::
+
+	<app_service_setting_name> : <service_name or id>
+
+In the above example, ``catalog_db`` is the name of the service in your :file:`app.py` and ``hydroexplorer-persistent`` is the name of the service on the portal. 
+
+.. tip::
+
+    Run ``tethys gen services`` to create a blank template of this file. By default the file will be saved to the current working directory.
+
+portal.yml 
+------------
+
+The file is designed to be maintained by the server administrator who can provide incoming apps with default services. 
+
+.. code-block:: python
+
+	# Portal Level Config File
+
+	version: 1.0
+	name: Tethys Main Portal
+	apps:
+	 hydroexplorer:
+	  services:
+	   persistent:
+	    catalog_db: test
+	   spatial:
+	   dataset:
+	   wps:
+
+
+**portal.yml Options:**
+
+* **version**: Indicated the version of the :file:`portal.yml` file. Current default : 1.0
+* **name**: Name of the portal
+
+* **apps/<app-name>/services/persistent** : List of persistent store settings in the app and the service to link to each.
+* **apps/<app-name>/services/dataset** : List of dataset settings in the app and the service to link to each.
+* **apps/<app-name>/services/spatial** : List of spatial persistent store settings in the app and the service to link to each.
+* **apps/<app-name>/services/wps** : List of Web Processing service settings in the app and the service to link to each. 
+
+Settings in each of the service sections above will need to be listed in the following format::
+
+	<app_service_setting_name> : <service_name or id>
+
+In the above example, ``catalog_db`` is the name of the service in your :file:`app.py` and ``test`` is the name of the service on the portal.
+
+.. tip::
+
+    Run ``tethys gen portal`` to create a blank template of this file. By default the file will be saved to ``$TETHYS_SRC/tethy_portal``.
+
+
+3. Restart Tethys Server
+==========================
+
+Restart tethys portal to effect the changes::
+
+    (tethys) $ tethys manage start
+
+.. tip::
+
+    Use the alias `tms` as a shortcut
+
+4. Configure App Settings
```

Comment:
This step is only needed if settings were not specified in the install process (either through the yaml file or interactively). We may need to take this step out, or rephrase it.

Reply:
Same as comment below.. I renamed this "Configure Additional App Settings" but I will look into the details

Reply:
@tylorbayer regarding configuring settings, if they weren't set in install this is how you do it manually.

Reply:
Resolved